.github/auto_assign.yml

```diff
@@ -0,0 +1 @@
+addAssignees: author
```

Comment:
one question, why separate files for config? can't we include this in the action?

Reply:
I tested that, and also looked at multiple repos. I haven to seen any around like that.  This is widely used GitHub action

Reply:
Okay, I see, I was looking into this action, found setting for reviewers, it would be nice to include reviewers as well to a PR by default

Reply:
@Gauravtalreja1 You mean reviewers as T1 and T2 by default ! Probably I wont because then people will be flooded with email every PR opened !